tools/airgap_linter.py

```diff
@@ -53,6 +53,7 @@ def is_bad_uri(uri, file_name):
         ".mesos:",
         "$MESOS_CONTAINER_IP",
         "{{FRAMEWORK_HOST}}",
+        "$FRAMEWORK_HOST",
```

Comment:
Looking at how these are matched to the URIs, that means that the following:
* `$MESOS_CONTAINER_IP_BUT_NOT_REALLY`
* `$FRAMEWORK_HOST_BUT_ALSO_NOT_QUITE`
will also be accepted as valid URIs.

Also, in terms of environment variables: `${FRAMEWORK_HOST}` and `${MESOS_CONTAINER_IP}` are also valid usages of environment variables.

Reply:
That is a good point. I will take a pass at writing a better URL evaluator.

I will add both of those alternate versions.

Reply:
(in a follow up)

Reply:
Ok. That's fine.